lib/ui/myplot/my_plot_detail_properties.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:farmsmart_flutter/model/crop_detail_property.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/strings.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:farmsmart_flutter/utils/styles.dart';
+import 'package:flutter/material.dart';
+
+class MyPlotDetailProperties {
+  Widget build(List<CropDetailProperty> cropProperties) {
+    return
+        ExpansionTile(
+          initiallyExpanded: true,
+            title: Text(Strings.myPlotDetailPropertiesTitle, style: Styles.detailTitleTextStyle()),
+            children: (cropProperties
+                .map((cropProperty) => buildPropertyItem(cropProperty))).toList());
+  }
+
+  Widget buildPropertyItem(CropDetailProperty cropProperty) {
```

Comment:
Does this mean that a crop will only have string based property (widgets)? Also is this the approach that allows hiding missing properties?

Reply:
Properties are gathered as a list of CropProperties.
We only add to this list, properties that have data.